test/test_stochastic.py

```diff
@@ -60,18 +60,18 @@ def test_stochastic_sag():
 
 def test_stochastic_asgd():
     # test asgd
-    n = 15
+    n_samples = 15
     reg = 1
-    numItermax = 100000
+    num_iter_max = 100000
     rng = np.random.RandomState(0)
 
-    x = rng.randn(n, 2)
-    u = ot.utils.unif(n)
+    x_samples = rng.randn(n_samples, 2)
```

Comment:
imho, I think `x` is fine or perhaps. `x_samples` and `n_samples` is perhaps too visually similar but denote very different objects.

Reply:
I agree that they are similar. I am fine with x, but I would keep n_samples to know what we are using here.

Reply:
Yes agreed for `n_samples`. `X` is probably more appropriate for multivariate data.